### PR TITLE
Fixes #3838: Adds ability to define a divider when declaring a menu item

### PR DESCRIPTION
**Pull request by ehelms**
from a plugin.

### CODE REVIEW

File: app/services/menu/divider.rb [hunk 1]
```diff
@@ -2,6 +2,7 @@ module Menu
   class Divider < Node
     def initialize(name, options={})
       @caption = options[:caption]
+      @parent = options.fetch(:parent, nil)
```
Comment (domcleal) [inline app/services/menu/divider.rb]: nitpick, options[:parent] works too as the default's nil

Comment (ehelms) [inline app/services/menu/divider.rb]: Are you saying you prefer that?

Comment (domcleal) [inline app/services/menu/divider.rb]: Sorry, I was :)  for consistency

**Comment by domcleal** [comment]
:+1: tested and added a note to http://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin

**Comment by domcleal** [comment]
Merged as f503738, thanks @ehelms!

**Pull request closed without merging**